src/containers/Header/Header.tsx

```diff
@@ -22,30 +27,80 @@ const getInternalLink = (singleClusterMode: boolean) => {
     return backend + '/internal';
 };
 
-function Header() {
-    const {singleClusterMode, header}: {singleClusterMode: boolean; header: HeaderItemType[]} =
-        useTypedSelector((state) => state);
+interface HeaderProps {
+    mainPage?: RawBreadcrumbItem;
```

Comment:
`mainPage` - additional breadcrumbs item for the extended versions (all clusters page)